frontends/web/src/components/aopp/vasp.tsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { h, JSX, RenderableProps } from 'preact';
+import * as styles from './vasp.css';
+import AOPPGroupLogo from '../../assets/exchanges/logos/aoppgroup.svg';
+import BitcoinSuisseLogo from '../../assets/exchanges/logos/bitcoin_suisse.png';
+import BittrLogo from '../../assets/exchanges/logos/bittr.png';
+import BityLogo from '../../assets/exchanges/logos/bity.png';
+import PocketBitcoinLogo from '../../assets/exchanges/logos/pocketbitcoin.svg';
+
+interface VASPProps {
+    fallback?: JSX.Element;
+    hostname: string;
+    prominent?: boolean;
+    withLogoText?: string;
+}
+
+const VASPLogoMap = {
+    'demo.aopp.group': AOPPGroupLogo,
+    'testing.aopp.group': AOPPGroupLogo,
+    'bitcoinsuisse.com': BitcoinSuisseLogo,
+    'bity.com': BityLogo,
+    'getbittr.com': BittrLogo,
+    'pocketbitcoin.com': PocketBitcoinLogo,
+};
+
+const VASPHostnameMap = {
+    'demo.aopp.group': 'AOPP.group',
+    'testing.aopp.group': 'AOPP.group',
+};
```

Comment:
For the AOPP demo, the visible domain name would be either `Aopp.group` or if we keep the subdomain: `Demo.aopp.group` or `Testing.aopp.group`.
Therefore I mapped both to `AOPP.group`.

Reply:
imho not needed? could just leave it as it was before, no need for this map. or am i missing sth?

Reply:
True but I'd assume we need name mapping anyways when some use a different subdomain i.e. `api.exchange.com` and now it is already prepared..

also this is a pure cosmetic change so our users don't see `Demo.aopp.group` or `Testing.aopp.group` nor `Aopp.group`